prow/plugins/releasenote/releasenote.go

```diff
@@ -205,14 +205,14 @@ func handlePR(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent) e
 	var comments []github.IssueComment
 	labelToAdd := determineReleaseNoteLabel(pr.PullRequest.Body)
 	if labelToAdd == releaseNoteLabelNeeded {
-		if !prMustFollowRelNoteProcess(gc, log, pr, prLabels, true) {
```

Comment:
>I think this use the same number of tokens as it did before? Which API call does this avoid?

@cjwagner this PR will avoid to call another `ListIssueComments` API (in `clearStaleComments`) when this condition is true.